src/test/java/org/jabref/logic/layout/format/AbstractParamLayoutFormatterTest.java

```diff
@@ -0,0 +1,88 @@
+package org.jabref.logic.layout.format;
+
+import java.util.List;
+
+import org.jabref.logic.layout.AbstractParamLayoutFormatter;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AbstractParamLayoutFormatterTest {
+    static class ParseArgumentTester extends AbstractParamLayoutFormatter {
+        public static List<String> callParseArgument(String arg) {
+            return parseArgument(arg);
+        }
+
+        @Override
+        public String format(String fieldText) {
```

Comment:
Why `String format(String)` and `void setArgument(String)` needs to be overrided?

Reply:
Ah, I see, the interface requires `void setArgument(String arg)`. But what about `format`?

Reply:
Same reason to `setArgument`. The interface `ParamLayoutFormatter` extends `LayoutFormatter`, which is also an interface requires `format` method.